src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -318,25 +324,30 @@ javac.opt.print=\
 javac.opt.printRounds=\
     Print information about rounds of annotation processing
 javac.opt.printProcessorInfo=\
-    Print information about which annotations a processor is asked to process
+    Print information about which annotations a processor\n\
+    is asked to process
 javac.opt.userpathsfirst=\
     Search classpath and sourcepath for classes before the bootclasspath instead of after
 javac.opt.prefer=\
-    Specify which file to read when both a source file and class file are found for an implicitly compiled class
+    Specify which file to read when both a source file and class file\n\
+    are found for an implicitly compiled class
 javac.opt.preview=\
-    Enable preview language features. To be used in conjunction with either -source or --release.
+    Enable preview language features. To be used in conjunction\n\
```

Comment:
I'd consider breaking "early", after the initial period.

Reply:
Fixed in b43a1ca1c6a.